client/src/Components/NavBar/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useContext, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Axios from 'axios';
+import { Layout, Typography, Menu } from 'antd';
+import AuthProvider from '../../Context/Authentication';
+import Public from './Public';
+import User from './User';
+import Button from '../Button';
+import UserInfo from '../UserInfo';
+
+const { Header } = Layout;
+const { Title } = Typography;
+
+const NavBar = () => {
+  const { setIsAuth, isAuth } = useContext(AuthProvider);
+  const history = useHistory();
+
+  const [isLoading, setLoading] = useState(false);
+  const [errMsg, setErrMsg] = useState('');
+  const [current, setCurrent] = useState('home');
+
+  const handleMenu = (e) => {
+    setCurrent(e.key);
+  };
+
+  const handleClick = async () => {
+    try {
+      setLoading(true);
+      await Axios('api/v1/logout');
+      setIsAuth(false);
+    } catch (err) {
+      setErrMsg('set interval error');
+    }
+  };
```

Comment:
Transfer the logout function to the userInfo component and use it directly instead of passing the handle logout function and loading as props